modules/wmi/testdata/v0.20.0/metrics.txt

```diff
@@ -103,6 +103,71 @@ process_start_time_seconds 1.667508736e+09
 # HELP process_virtual_memory_bytes Virtual memory size in bytes.
 # TYPE process_virtual_memory_bytes gauge
 process_virtual_memory_bytes 3.5569664e+07
+# HELP windows_adcs_challenge_response_processing_time_seconds Last time elapsed for challenge response
+# TYPE windows_adcs_challenge_response_processing_time_seconds gauge
+windows_adcs_challenge_response_processing_time_seconds{cert_template="Administrator"} 0
+windows_adcs_challenge_response_processing_time_seconds{cert_template="DomainController"} 0
+windows_adcs_challenge_response_processing_time_seconds{cert_template="_Total"} 0
+# HELP windows_adcs_challenge_responses_total Total certificate challenge responses processed
+# TYPE windows_adcs_challenge_responses_total counter
+windows_adcs_challenge_responses_total{cert_template="Administrator"} 0
+windows_adcs_challenge_responses_total{cert_template="DomainController"} 0
+windows_adcs_challenge_responses_total{cert_template="_Total"} 0
+# HELP windows_adcs_failed_requests_total Total failed certificate requests processed
+# TYPE windows_adcs_failed_requests_total counter
+windows_adcs_failed_requests_total{cert_template="Administrator"} 0
+windows_adcs_failed_requests_total{cert_template="DomainController"} 0
+windows_adcs_failed_requests_total{cert_template="_Total"} 0
+# HELP windows_adcs_issued_requests_total Total issued certificate requests processed
+# TYPE windows_adcs_issued_requests_total counter
+windows_adcs_issued_requests_total{cert_template="Administrator"} 0
+windows_adcs_issued_requests_total{cert_template="DomainController"} 1
+windows_adcs_issued_requests_total{cert_template="_Total"} 1
+# HELP windows_adcs_pending_requests_total Total pending certificate requests processed
+# TYPE windows_adcs_pending_requests_total counter
+windows_adcs_pending_requests_total{cert_template="Administrator"} 0
+windows_adcs_pending_requests_total{cert_template="DomainController"} 0
+windows_adcs_pending_requests_total{cert_template="_Total"} 0
```

Comment:
_Total is not needed. Why did you leave it?

Reply:
Should be filtered out in the code. Cloud UI will show aggregated counters.

Reply:
Because I am still learning cloud features. :man_facepalming: 
I removed it.